### PR TITLE
Add TOUCH_MODE_ACTIONBAR (refactored)

### PR DESCRIPTION
**Pull request by lexer**
This is a custom touch mode that allows open sliding menu only by sliding of actionbar.

Actionbar size now is part of MenuDrawer styles.
Statusbar height is obtained dynamically using fitSystemWindows.

### CODE REVIEW

**Comment by SimonVT** [comment]
I think there should be a more generic term for this than TOUCH_MODE_ACTIONBAR. It should also be possible to set the touch area at the top/bottom for left/right drawer, and left/right for top/bottom drawer.
Let me know if you're up for it, otherwise I can take a look at it myself this weekend.

**Comment by lexer** [comment]
Could you propose naming and what type of parameter we will have for this?

File: library/src/net/simonvt/menudrawer/MenuDrawer.java [hunk 1]
```diff
@@ -55,6 +55,11 @@
     private static final int DEFAULT_DRAG_BEZEL_DP = 24;
 
     /**
+     * The default touch bezel size of the drawer in dp.
+     */
+    private static final int DEFAULT_ACTIONBAR_HEIGHT_DP = 48;
```
Comment (JakeWharton) [inline library/src/net/simonvt/menudrawer/MenuDrawer.java:60]: This is wrong. In landscape it's 40dp and on sw600dp devices it's 56dp. You can't rely on this value.

Comment (JakeWharton) [inline library/src/net/simonvt/menudrawer/MenuDrawer.java:60]: That is, if I use this library without adding the theme attribute the draggable area will be incorrect on those alternate configurations.

File: samples/res/values/styles.xml [hunk 2]
```diff
@@ -34,6 +34,7 @@
     <style name="MenuDrawerStyle" parent="Widget.MenuDrawer">
         <item name="mdActiveIndicator">@drawable/menu_arrow</item>
         <item name="mdMenuSize">250dp</item>
+        <item name="mdActionBarSize">?android:attr/actionBarSize</item>
```
Comment (JakeWharton) [inline samples/res/values/styles.xml:37]: API 11+ only. This should be in `values-v11/styles.xml`.

**Comment by SimonVT** [comment]
Thanks, but I'm gonna close this in favor of something completely different, yet similar. Stay tuned. :)

**Pull request closed without merging**